### ISSUE TITLE
Wordpress database error on table 'wp_woocommerce_multisafepay'

### ISSUE THREAD

**Issue by Alessandro-4EYES** [MultiSafepay/WooCommerce#8]
Hello,
We are seeing this error in the logs of our site every 1-2 hours. 
We did install version 3.5.0 last week.

```
[04-May-2020 00:19:27 UTC] WordPress database error Table 'wp_woocommerce_multisafepay' already exists for query CREATE TABLE wp_woocommerce_multisafepay
                                (   id bigint(20) NOT NULL auto_increment,
                                    trixid varchar(200) NOT NULL,
                                    orderid varchar(200) NOT NULL,
                                    status varchar(200) NOT NULL,
                                    PRIMARY KEY  (id)
                                ) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_520_ci made by require('wp-blog-header.php'), require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/multisafepay/multisafepay.php'), MultiSafepay\WooCommerce\Gateways::register, dbDelta, QM_DB->query
```

**Comment by Ronald-MultiSafepay** [comment]
@Alessandro-4EYES Thank you for reporting this. We will investigate this issue and give you an update as soon as possible.

**Comment by Ronald-MultiSafepay** [comment]
@Alessandro-4EYES 
This issue has been resolved and will be available in the next release (expected end of the month)
The errors in the log-file have no impact on the correct working of the plugin, it's just a message.

**Comment by Ronald-MultiSafepay** [comment]
@Alessandro-4EYES 
We just released v3.5.2 of our WooCommerce plugin.
This release includes a fix for the issue you mentioned.
